packages/core/strapi/lib/services/entity-service/index.d.ts

```diff
@@ -19,8 +21,12 @@ type PaginationInfo = {
   total: number;
 };
 
-type Params<T> = {
-  fields?: (keyof T)[];
+type DataParam<TSchemaUID extends Common.UID.Schema> = {
+  [Key in Attribute.GetKeys<TSchemaUID>]?: AttributeUtils.GetValue<Attribute.Get<TSchemaUID, Key>>;
+};
+
+type Params<TSchemaUID extends Common.UID.Schema> = {
```

Comment:
This isn't the right type to update :)

packages/core/strapi/lib/services/entity-service/types/params/index.d.ts

Reply:
```
export type For<TSchemaUID extends Common.UID.Schema> = {
  sort?: Sort.Any<TSchemaUID>;
  fields?: Fields.Any<TSchemaUID>;
  filters?: Filters.Any<TSchemaUID>;
  data?: Data.Any<TSchemaUID>;
} & Pagination.Any;
```